tests/pass-dep/libc/prctl.rs

```diff
@@ -0,0 +1,47 @@
+//@only-target: android # prctl is currently supported for Android only
+use libc::c_char;
+use std::ffi::CStr;
+use std::ffi::CString;
+
+fn main() {
+    let long_name = CString::new(
+        std::iter::once("test_named_thread_truncation")
+            .chain(std::iter::repeat(" yada").take(100))
+            .collect::<String>(),
+    )
+    .unwrap();
+
+    // prctl supports thread names up to 16 characters includding nul.
+    // And since the input string is longer it should fail.
```

Comment:
Typo and suggestion: 
```suggestion
    // prctl supports thread names up to 16 characters including null.
    //  Since the input string is longer, it should fail.
```